packages/connect/src/data/firmwareInfo.ts

```diff
@@ -32,7 +44,52 @@ export const parseFirmwareReleases = (json: any, deviceModel: DeviceModelInterna
     });
 };
 
-export const getReleases = (deviceModel: DeviceModelInternal) => releases[deviceModel] || [];
+export const parseFirmwareReleasesRemote = (json: any, deviceModel: DeviceModelInternal) => {
+    Object.keys(json).forEach(key => {
+        const release = json[key];
+        remoteReleases[deviceModel]?.push({
+            ...release,
+        });
+    });
+};
+
+export const getReleases = (deviceModel: DeviceModelInternal) => {
+    const latestReleases = remoteReleases[deviceModel];
+    if (latestReleases && latestReleases.length > 0) {
+        return latestReleases;
+    }
+    return releases[deviceModel] || [];
+};
+
+export const parseMessageRelease = async (): Promise<any> => {
+    const url = path.join(
+        __dirname,
+        '../../..',
+        'suite-common/message-releases/releases/test-releases.v1.json',
+    );
```

Comment:
This is just for Dev, this will be downloaded from data.trezor.io and it will be a signed file.

Reply:
We will most likely also bundle this file, so it can be used offline, and use it always to get the firmware to update. The current releases.json still will be necessary but we will use it for other things like the languages supported in older versions and other information from older FWs.